pkg/userdata/rockylinux/testdata/kubelet-v1.24-aws.yaml

```diff
@@ -241,6 +241,8 @@ write_files:
       --lock-file=/tmp/kubelet.lock \
       --container-runtime=remote \
       --container-runtime-endpoint=unix:///run/containerd/containerd.sock \
+      --dynamic-config-dir=/etc/kubernetes/dynamic-config-dir \
+      --feature-gates=DynamicKubeletConfig=true \
```

Comment:
This shouldn't happen, DynamicKubeletConfig been deleted in 1.24

Reply:
I somehow messed up the constraint 🤦 Updated the PR.